src/bootstrap/builder.rs

```diff
@@ -751,6 +751,11 @@ impl<'a> Builder<'a> {
         cmd
     }
 
+    /// Gets a path to the jsondocck tool
+    pub fn jsondocck(&self, compiler: Compiler, target: TargetSelection) -> PathBuf {
+        self.ensure(tool::JsonDocCk { compiler, target })
+    }
+
```

Comment:
This is small enough I would rather just call `self.ensure` inline, that would also make it more clear it builds `JsonDocCk` if necessary.

Reply:
Hm. I'd be alright with that, I added this for 'conformity', there are a bunch of these functions for things like 'rustc' or 'rustdoc'. On the other hand, those things are ensured in more than one place :P

Reply:
I would rather inline it, yeah - there are a *lot* of tools and I don't think they all need their own function.